mapbox_tilesets/scripts/cli.py

```diff
@@ -27,6 +27,15 @@ def _get_api():
     return os.environ.get("MAPBOX_API", "https://api.mapbox.com")
 
 
+def _get_session(
+    application=mapbox_tilesets.__name__, version=mapbox_tilesets.__version__
+):
+    """Get a configured session"""
+    s = Session()
+    s.headers.update({"user-agent": "{}/{}".format(application, version)})
```

Comment:
does capitalization matter here?

Reply:
Officially nope! Header names are case-insensitive according to the HTTP 1/2 specs.